server/data/orchestrationApiClient.test.ts

```diff
@@ -470,6 +470,22 @@ describe('orchestrationApiClient', () => {
     })
   })
 
+  describe('isBlockedDate', () => {
+    it('should return excludeDateDto if exclude date found for passed date', async () => {
+      const results = TestData.isExcludeDateDto()
```

Comment:
Suggest not bothering with this `TestData` method and just putting `true` instead of `results` on line 481